zzbot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  joint1_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint1
    pid: {p: 0.02, i: 0.02, d: 0}

  joint2_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint2
    pid: {p: 0.02, i: 0.02, d: 0}

  joint3_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: joint3
    pid: {p: 0.02, i: 0.02, d: 0}